docs/run-platform-server/ingestion-filtering.mdx

```diff
@@ -3,89 +3,61 @@ title: Ingestion Filtering
 order: 46
 ---
 
-The Ingestion Filtering feature is now released for public beta testing available from Horizon [version 2.18.0](https://github.com/stellar/go/releases/tag/horizon-v2.18.0) and up.
-
 ## Overview
 
-Ingestion Filtering enables Horizon operators to drastically reduce storage footprint of their Horizon DB by whitelisting Assets and/or Accounts that are relevant to their operations. This feature is ideally suited for private Horizon operators who do not need full history for all assets and accounts on the Stellar network.
+Ingestion Filtering enables Horizon operators to drastically reduce the storage footprint of the historical data in the Horizon database by white-listing Assets and/or Accounts that are relevant to their operations.
```

Comment:
can you rebase your commits on top of partner_experience, your branch has https://github.com/stellar/stellar-docs/pull/168 locally and it's triggering unrelated diffs in this PR, https://github.com/stellar/stellar-docs/pull/168 was merged to partner_experience, thanks!

Reply:
done